opt/cs50/bin/flask

```diff
@@ -46,24 +36,11 @@ if [[ "$1" == "run" ]]; then
     # using regex to handle -pxxxx, -p xxxx, --port xxxx, --port=xxxx
     fuser --kill -INT "${port//[^0-9]}/tcp" &> /dev/null
 
-    # Check whether online (or offline)
-	if [ "$CS50_IDE_TYPE" == "online" ]; then
-		URL="https://$C9_HOSTNAME"
-	else
-		URL="http://localhost"
-	fi
-
-    # If $FLASK_APP isn't set, check for application.py (for backwards compatibility), else default to app.py
-    if [[ -z "$FLASK_APP" ]]; then
-        if [[ -f "application.py" ]]; then
-            FLASK_APP="application.py"
-        else
-            FLASK_APP="app.py"
-        fi
-    fi
-
     # Spawn flask
-    unbuffer /usr/local/bin/flask run $host $port $reload $threads $options | sed "s#\(.*\)http://[^:]\+\(:.\+\)#\1$URL\2#"
+    FLASK_APP="$FLASK_APP" \
+    FLASK_DEBUG="${FLASK_DEBUG:-0}" \
+    FLASK_ENV="development" \
```

Comment:
These 3 lines aren't needed?

Reply:
Sorry, accidentally removed a bit of code, which defines some of those. The full context is https://github.com/cs50/cli/blob/172aa6b82bafb2143952f27a048d6394ccad60c2/opt/cs50/bin/flask.